### DISCUSSION TITLE
How do we memory profile a .NET MAUI application

### DISCUSSION THREAD

**Discussion by cosminstirbu** [dotnet/maui#11813] [Q&A]
Hello,

How do we memory profile a .NET MAUI application?

In the past we were using Xamarin Profiler, however, in Visual Studio for Mac Enterprise 17.4 I can't even find the option to run the Xamarin Profiler.

What is the recommended way of doing that?

**Comment by nk54** [comment]
The profiler is available in "Analyse" > "Xamarin profiler"

But it doesn't work for MAUI as far as I know
